tests/mpi/refine_and_coarsen_fixed_number_07.cc

```diff
@@ -0,0 +1,187 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2009 - 2014 by the deal.II authors
```

Comment:
It is copied from refine_and_coarsen_fixed_number_02.
Other tests may also need update.